### ISSUE TITLE
Weibull parameters as y_train?

### ISSUE THREAD

**Issue by NataliaVConnolly** [ragulpr/wtte-rnn#22]
Hi Egil,

   I was just wondering if there are any advantages to finding the Weibull parameters through an RNN activation layer vs. first fitting the data to the Weibull to get the (alpha, beta) params and giving those to the RNN loss function instead of TTE and censored indicators?  So basically separating out (alpha, beta) fitting and optimization instead of doing then jointly in the RNN.    What do you think?

**Comment by ragulpr** [comment]
Hi there,
I've been thinking about something similar. There's a big problem with having the correct initialization parameters of the output layer, but it also needs to be simple. Training twice feels hacky. At present initialize the output layer instead to be reasonable.

It will then typically figure out the rest in the few first iterations.  Here *reasonable* is thought up as when we fix beta to 1 and alpha around mean expected tte.

Did you have problems with initialization?

Also check out `test_output_lambda_initialization()` in `tests/test_keras.py`
